### PR TITLE
update kubernets version dependency

### PR DESCRIPTION
**Pull request by dominikholler**
@dhiller as you suggested in https://groups.google.com/g/kubevirt-dev/c/zXLSZD9m8Jg/m/RHPVCgkjCQAJ .

### CODE REVIEW

**Comment by cwilkers** [comment]
/lgtm
@dhiller I'll leave it to you to approve the wording, I think this was your suggestion.

File: docs/operations/installation.md [hunk 1]
```diff
@@ -11,8 +11,9 @@ virtual machines.
 A few requirements need to be met before you can begin:
 
 -   [Kubernetes](https://kubernetes.io) cluster or derivative
-    (such as [OpenShift](https://github.com/openshift/origin), Tectonic)
-    based on Kubernetes 1.10 or greater
+    (such as [OpenShift](https://github.com/openshift/origin))
+    based on a one of the latest three Kubernetes releases that are
```
Comment (dhiller) [inline docs/operations/installation.md:15]:
```suggestion
    based on one of the latest three Kubernetes releases that are
```

**Review by dhiller — COMMENTED** [review]
One nit inside, otherwise looks fine, thanks!

/approve

**Comment by mazzystr** [comment]
/lgtm
/approve

Thanks for that update

**Pull request merged**